cypress/integration/my_patches.js

```diff
@@ -1,23 +1,46 @@
 /// <reference types="Cypress" />
 
 const MY_PATCHES_ROUTE = "/my-patches";
-describe("my patches page", function () {
-  beforeEach(() => {
-    cy.server();
+describe("My Patches Page", () => {
+  before(() => {
     cy.login();
   });
 
-  it("Should render with Show Commit Queue box checked when commitQueue not indicated in URL query param", () => {
-    cy.visit(MY_PATCHES_ROUTE);
-    cy.get("[data-cy=commit-queue-checkbox]").should("be.checked");
+  beforeEach(() => {
+    cy.preserveCookies();
+    cy.listenGQL();
   });
 
-  it("Should render with Show Commit Queue box unchecked when commitQueue is false in URL query param", () => {
-    cy.visit(`${MY_PATCHES_ROUTE}?commitQueue=false`);
-    cy.get("[data-cy=commit-queue-checkbox]").should("not.be.checked");
+  it("Initial load should make a userPatches gql query with default query params when none are specified in the URL", () => {
+    cy.visit(MY_PATCHES_ROUTE);
+    cy.waitForGQL("userPatches", {
+      "request.body.variables.$includeCommitQueue": (v) => v == true,
+      "request.body.variables.$limit": (v) => v === 10,
+      "request.body.variables.$page": (v) => v === 0,
+      "request.body.variables.$patchName": (v) => v === "",
+      "request.body.variables.$statuses": (v) =>
+        Array.isArray(v) && v.length === 0,
+    }).then((xhr) => console.log(xhr));
   });
-  it("Should render with Show Commit Queue box checked when commitQueue is true in URL query param", () => {
-    cy.visit(`${MY_PATCHES_ROUTE}?commitQueue=true`);
-    cy.get("[data-cy=commit-queue-checkbox]").should("be.checked");
+
+  describe("Show commit queue checkbox", () => {
+    beforeEach(() => {
+      cy.preserveCookies();
+    });
+
+    it("Should render with Show Commit Queue box checked when commitQueue not indicated in URL query param", () => {
```

Comment:
tests do not need to start with "should render with". For example, the description can be changed to "Commit queue checkbox". and then this test name would be "Renders checkbox checked by default". Longer test names are not always more clear than short, simple ones.

Reply:
This test entered the codebase from a previous PR.